tests/node_tests/backbeat/tests/retry/retry.js

```diff
@@ -20,14 +20,17 @@ const keyPrefix = `${srcBucket}/${hex}`;
 const key = `object-to-replicate-${Date.now()}`;
 const REPLICATION_TIMEOUT = 1200000;
```

Comment:
20-min timeout O.O

Reply:
I believe this is leftover from running the test suite without updating the exponential backoff in backbeat's config. The goal is to not create tests that take so long (and instead use a custom backbeat image with an updated config) so I've reset this to 5 minutes like the other replication timeout constants.